core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.html

```diff
@@ -15,7 +15,7 @@
         class="dot-edit__toolbar"
         [pageState]="pageState"
         [variant]="variantData | async"
-        [runningExperiment]="runningExperiment$ | async"
+        [runningExperiment]="pageState.state.runningExperiment"
```

Comment:
Ohhh this is part of the page state now.

Reply:
yea the other option was to make multiple request, to check if disable the layout menu also to show/hide the running indicator at the top of the edit mode and any other restriction that can happen based on experiments status.